.bashrc

```diff
@@ -156,4 +156,13 @@ if [ ! -d "{$hostdir}/bin" ]
 then
     mkdir -p "${hostdir}/bin" 
 fi
-export PATH="${PATH}:${hostdir}/bin"
\ No newline at end of file
+export PATH="${PATH}:${hostdir}/bin"
+
+# To allow persistent vimrc modifications,
+# copy from .vimrc on host to ~/.vimrc
+# and install plugins in subshell in background, suppressing output
+hostvimrc="${hostdir}/.vimrc"
+if [ -f "$hostvimrc" ]; then
+    cp "$hostvimrc" ~/.vimrc
+    (vim -c -es 'PluginInstall' -c -es 'qa!' </dev/null >/dev/null 2>&1 &)
```

Comment:
nit: spaces

Reply:
iirc, if I make a commit fixing this you'll have to re-approve, so I'll merge this now and fix later (TM).